work_queue/src/work_queue_server.c

```diff
@@ -0,0 +1,197 @@
+#include "work_queue_json.h"
+#include "link.h"
+#include "jx.h"
+#include "jx_print.h"
+#include "jx_parse.h"
+
+#include <string.h>
+#include <time.h>
+
+char * workqueue = "{ \"name\" : \"server_wq\" , \"port\" : 1234 }";
+
+void reply(struct link *client, char *method, char* message, int id){
+
+    struct jx_pair *jsonrpc, *result, *idd;
+    struct jx *j;
+    char *response;
+    time_t t;
+
+    idd = jx_pair(jx_string("id"), jx_integer(id), NULL);
```

Comment:
Get in the habit of combining the variable declaration with initial assignment.
It shortens things and makes it clearer what the scope of each declaration is.